Gopkg.toml

```diff
@@ -18,3 +18,7 @@
 [[constraint]]
   name = "github.com/pkg/errors"
   version = "0.8.0"
+
+[[constraint]]
+  name = "github.com/boltdb/bolt"
+  version = "^1.0.0"
```

Comment:
nit: don't use the caret. this is a confusing area already, we should be consistent in how we do.